engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java

```diff
@@ -276,12 +297,39 @@ public void doHttpRequest(Event event, Endpoint endpoint, HttpRequest<Buffer> re
                 details.put("error_message", e.getMessage()); // TODO This message isn't always the most descriptive..
                 history.setDetails(details.getMap());
             }
+            updateMetrics(history.getStatus(), isEmailEndpoint);
```

Comment:
We talked about counting successes (instead of processed events) and failures from a `finally` block depending on the notification status. Is that no longer part of the plan?